### ISSUE TITLE
Enable Hot Reloading

### ISSUE THREAD

**Issue by glenbraun** [DemocracyLab/CivicTechExchange#477]
Currently we need to run npm run build/dev in order to build front-end changes. If builds happened automatically when changes are made, it would streamline the development process, especially for development environments like Docker where running the build command is more laborious.

https://trello.com/c/G3LA9fka

**Comment by PeterBreen** [comment]
For scope considerations, on this, just concentrate on Docker for now (if that's even relevant?) -- Also see #927 as another issue on the same problem, we just didn't catch it as a dupe.

**Comment by AM-77** [comment]
Hello guys, I just open a PR for this, can anyone give it a look please.

also I want to apologies to @Yyao5 for taking the ticket without leaving a comment or self-assign it.

I can not self-assign issues, I think because I am not a member in this org yet, can anyone add me please.

thank you

**Comment by Yyao5** [comment]
That’s all right. I was working on this with Abhishek. I’m not sure if you have writing access to the repo but if not, maybe Marlon can add you to it. I just assigned you to the issue.Thanks,YutongOn Aug 19, 2023, at 6:28 AM, Mohamed Amine ***@***.***> wrote:﻿
Hello guys, I just open a PR for this, can anyone give it a look please.
also I want to apologies to @Yyao5 for taking the ticket without leaving a comment or self-assign it.
I can not self-assign issues, I think because I am not a member in this org yet, can anyone add me please.
thank you

**Comment by Yyao5** [comment]
Current related PR can reflect changes with automatic webpack recompilation and after manually refreshing the browser. Further attention is required to enable reloading in the browser automatically.
